### PR TITLE
Feature/docs logo

### PR DESCRIPTION
**Pull request by hannahker**
Making things look fancier!

- Updating the styling of our docs to include Centre, OCHA, and MapAction (where appropriate) logos
- Adding in a Centre favicon and standardizing the format of the browser title
- Adding custom styling to better match Centre branding (font, colour, etc)
- Updating paths where needed to be able to regenerate the various `.Rmd` files

Looks something like this... 

Capture

### CODE REVIEW

**Comment by Tinkaa** [comment]
Woohoo I love this! 
Two small comments but you can merge
Also, did you have a vision on when to add the MapAction logo? I saw for example for mwi that you only added it for the impact analysis?

**Review by Tinkaa — APPROVED** [review]

**Comment by Tinkaa** [comment]
sorry Hannah, apparently didn't "finish my review"
Regarding MapAction logo: sounds good to me

File: analyses/country_template/docs/style.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+body{
+  font-family: 'Source Sans Pro';
+}
+/* Headers */
+h1{
```
[the remark this answers is not in the document]
Comment (hannahker) [reply analyses/country_template/docs/style.css:5]: To keep things simpler, I've just adjusted styling for h1, but would be happy to adjust further if there's an official Centre style guide that we can rely on. I'll merge for now as is but there can definitely be future improvements down the line!

File: analyses/mwi/docs/mwi_dry_spells_indicator_analyses.Rmd [hunk 2]
```diff
@@ -15,6 +17,13 @@ biblio-style: "apalike"
 link-citations: true
 ---
 
+<br> 
+<br>
+<br>
+
+# Forecasting dry spells in Malawi
+Analysis last updated `r Sys.Date()`
```
[the remark this answers is not in the document]
Comment (hannahker) [reply analyses/mwi/docs/mwi_dry_spells_indicator_analyses.Rmd]: Good point! Have adjusted this and removed the automatic date update from the template.

**Pull request merged**